lib/Experian/IDAuth.pm

```diff
@@ -116,9 +115,11 @@ sub save_pdf_result {
 sub has_downloaded_pdf {
     my $self     = shift;
     my $file_pdf = $self->_pdf_report_filename;
-    -e ($file_pdf) || return;
-    my $file_type = qx(file $file_pdf);
-    return $file_type =~ /PDF/;
+    return 0 unless -e $file_pdf;
+    open my $fh, '<', $file_pdf;
```

Comment:
```perl
open ... or croak "Could not open PDF: $!";
```

There may be time when file exists, but unreadable for some reason (e.g. disk issues.)